spec/services/deployment_service_spec.rb

```diff
@@ -335,25 +335,26 @@
   end
 
   describe '.restart_service' do
-    let(:mock_adapter) { double('adapter', delete_pods: true) }
+    let(:mock_adapter) { double('heee', patch_deployment: true) }
```

Comment:
seems like an odd name to give to the double?

Reply:
Woops, debugging.  Fixed now.